### ISSUE TITLE
Model Improvement

### ISSUE THREAD

**Issue by AFKafe** [MelanX/botanical-machinery#74]
### Describe your idea

Greetings,

I would like to provide my opinion regarding the 3D models of your machines.

Above all, I really like your machines. I prefer the “Mechanical Daisy” without the frame, but they don't all have the same visual. 

Being in the process of creating a pack I said to myself that I would try to modify the visual of the frames of your machines to give them the appearance of Alfsteel from Mythic Botany. At the same time I tried to remove the frame to leave only the part on the ground (like the Mechanical Daisy) but I am facing a HitBox problem. By removing the frame, its HitBox is still the same and after verification there is no way to change it other than in the mod code.

![image](https://github.com/user-attachments/assets/31a69d13-f66f-47f2-9eaa-8d079707df60)
![image](https://github.com/user-attachments/assets/99c8faa7-dee1-4a04-a7d5-7f7f0c33b75c)
![image](https://github.com/user-attachments/assets/f7a470cb-0878-481a-87cc-94374076d860)

After this long explanation, I come to my question, would you consider having a visual similar to the "Mechanical Daisy" for all your machines or is there no desire to change the visual for now?

Thanks in advance,
AFKafé

**Comment by MelanX** [comment]
I really like your idea, but I feel a bit uncomfortable with it. I mean, the Daisy is the only one which doesn't need mana. Everything else needs mana and is more like a "machine" in my mind. That's why it should have some differences, but to be honest, I don't like the frames anymore, too. I need to find something else. I also thought about filling the frames with mana glass. At least the old version of mana glass was beautiful, I don't know whether there's already a new texture. That would also solve some problems with mana spreaders don't hit the blocks properly.

tl;dr I want to change the visuals but I don't know how exactly. I don't think that it would be like your idea, sorry. Maybe the glass idea is fine for you, too. At least, it would change the bounding box to a cube :)

**Comment by AFKafe** [comment]
I like the idea of ​​the Mana Glass which will close the "machine". And it will keep the very nice visual of your work. (And if it solves the hitbox problems for Mana Spreaders, even better)

Thanks for this, I continue to enjoy your work and look forward to integrating it into the modpack I'm working on.
